### ISSUE TITLE
Download failure: Sat Sep 10 18:04:17 UTC 2016

### ISSUE THREAD

**Issue by MXEBot** [mxe/mxe#1519]
See http://sprunge.us/EWAN?log.

**Comment by starius** [comment]
levmar's website seem to block automated downloads.

```
$ wget http://users.ics.forth.gr/~lourakis/levmar/levmar-2.6.tgz
2016-09-10 19:36:24 ERROR 500: Internal Server Error.

$ wget -U 'Mozilla/5.0 (Windows NT 6.1; rv:45.0) Gecko/20100101
Firefox/45.0' http://users.ics.forth.gr/~lourakis/levmar/levmar-2.6.tgz
HTTP request sent, awaiting response... 200 OK
2016-09-10 19:36:28 (149 KB/s) - `levmar-2.6.tgz' saved [81143/81143]
```

**Comment by tonytheodore** [comment]
fixed in #1661
